annotations/transport/annotations_router.go

```diff
@@ -234,3 +241,42 @@ func decodeUpdateAnnotationRequest(r *http.Request) (*influxdb.AnnotationCreate,
 
 	return u, nil
 }
+
+func storedAnnotationToEvent(s *influxdb.StoredAnnotation) (*influxdb.AnnotationEvent, error) {
```

Comment:
These two functions for converting stored annotations to events are here for now, but I'll probably move them when I make the storage service implementation, since there will be other similar functions needed there.